packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -591,6 +591,7 @@ export async function getLatestSnapshotInfo(
 	);
 }
 
+/* eslint-disable @fluid-internal/fluid/no-hyphen-after-jsdoc-tag -- false positive */
```

Comment:
Just curious, what line is it triggering on here? The plugin has tests for the `@param` case, so I don't expect this comment to be a problem (even with the known bugs).

Reply:
good catch - this was not needed.